app/_includes/md/deb.md

```diff
@@ -0,0 +1,45 @@
+## Installation
+
+Kong can be installed by downloading an installation package or from our apt repository
+
+{% navtabs %}
+{% navtab Packages %}
+
+{% if include.distribution == "ubuntu" %}
+
+- [Xenial]({{ site.links.download }}/gateway-community/ubuntu/xenial/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [Bionic]({{ site.links.download }}/gateway-community/ubuntu/bionic/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [Focal]({{ site.links.download }}/gateway-community/ubuntu/focal/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+
+{% endif %}
+
+{% if include.distribution == "debian" %}
+
+- [8 Jessie]({{ site.links.download }}/gateway-community/debian/jessie/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [9 Stretch]({{ site.links.download }}/gateway-community/debian/stretch/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [10 Buster]({{ site.links.download }}/gateway-community/debian/buster/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [11 Bullseye]({{ site.links.download }}/gateway-community/debian/bullseye/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+
+{% endif %}
+
+To install from the command line
+
+```bash
+$ curl -Lo kong.{{site.data.kong_latest.version}}.amd64.deb "{{ site.links.download }}/gateway-community/$(lsb_release -is)/$(lsb_release -cs)/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb"
+$ sudo dpkg -i kong.{{site.data.kong_latest.version}}.amd64.deb
+```
+
+{% endnavtab %}
+{% navtab Repository %}
+
+**APT Repositories**
+
+```bash
+$ sudo apt-add-repository -u "deb [allow-insecure=yes] {{ site.links.download }}/gateway-community/$(lsb_release -is)/$(lsb_release -sc)/ default all"
```

Comment:
this needs to be changed to `echo .... | tee /etc/apt/sources.list`

Reply:
@hutchic I'm not sure what needs to be replaced here exactly- are you saying the entire code block starting on line 38 should be: `echo sudo apt-add-repository -u "deb [allow-insecure=yes] {{ site.links.download }}/gateway-community/$(lsb_release -is)/$(lsb_release -sc)/ default all" | tee /etc/apt/sources.list` ?

Reply:
It was more a note to myself. I've updated to reflect production 👍